res/values/strings.xml

```diff
@@ -699,6 +703,13 @@
 
     <!-- ImageEditorHud -->
     <string name="ImageEditorHud_draw_anywhere_to_blur">Draw anywhere to blur</string>
+    <string name="ImageEditorHud_add_text">Add text</string>
+    <string name="ImageEditorHud_blur">Blur</string>
+    <string name="ImageEditorHud_brush_marker">Marker brush</string>
+    <string name="ImageEditorHud_brush_highlight">Highlight brush</string>
+    <string name="ImageEditorHud_crop">Crop</string>
+    <string name="ImageEditorHud_flip">Flip</string>
+    <string name="ImageEditorHud_rotate">Rotate</string>
```

Comment:
BTW is it possible for blind people to use the image editor? If not, do we need these?

Reply:
there are people that are not blind but can't see well, they might use the image editor and not see the details of the icons or wonder what the icons mean, for example the markers icons are almost identical, etc.

Reply:
Makes sense